readthedocs/settings/base.py

```diff
@@ -682,7 +682,10 @@ def DOCKER_LIMITS(self):
     DEFAULT_PRIVACY_LEVEL = 'public'
     DEFAULT_VERSION_PRIVACY_LEVEL = 'public'
     ALLOW_ADMIN = True
+
+    # Organization settings
     RTD_ALLOW_ORGANIZATIONS = False
+    ORG_DEFAULT_SUBSCRIPTION_PLAN_SLUG = 'trial-v2-monthly'
```

Comment:
What is `ORG` here? Organization? Not the best setting name.

Reply:
This is the setting already exists on .com, don't want to break ops. But I think this is the only place where this would need to be changed?

Reply:
Yea, I think it's fine to leave, but it's confusing in this context where `org` could mean `Organization` or `readthedocs.org`.